deploy/charts/cert-manager/templates/_helpers.tpl

```diff
@@ -43,9 +43,11 @@ Create the name of the service account to use
 
 {{/*
 Expand the name of the chart.
+Manually fix the 'app' and 'name' labels to 'webhook' to maintain
+compatibility with the v0.9 deployment selector.
 */}}
 {{- define "webhook.name" -}}
-{{- printf "%s-webhook" (default .Chart.Name .Values.nameOverride) | trunc 63 | trimSuffix "-" -}}
+{{- printf "webhook" | trunc 63 | trimSuffix "-" -}}
```

Comment:
This change is required as the `spec.selector` field on Deployments is immutable - we moved the webhook from its own chart into the main one, which caused this template to change, thus breaking upgrades (requiring people to `kubectl delete` the deployment before upgrading, or borking users Helm releases).